packages/playground/react-native.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  // Uncomment the below when running autolink-windows for playground-win32
```

Comment:
when would that be the case? e.g. is autolinking something people need to do when they do XYZ? (what is XYZ?)

Reply:
Fixed.